firmware/tunerstudio/rusefi.input

```diff
@@ -2498,14 +2507,14 @@ cmd_set_engine_type_default					= "w\x00\x31\x00\x00"
 		field = "Two-wire mode",						etb_use_two_wires, {throttlePedalPositionAdcChannel != 16}
 		field = "PWM Frequency",						etbFreq, {throttlePedalPositionAdcChannel != 16}
 		field = etbCalibrationOnStart,					etbCalibrationOnStart
-		field = "No1 Direction #1",						etb1_directionPin1, {throttlePedalPositionAdcChannel != 16}
-		field = "No1 Direction #2",						etb1_directionPin2, {throttlePedalPositionAdcChannel != 16}
-		field = "No1 Control #1",						etb1_controlPin1, {throttlePedalPositionAdcChannel != 16 && etb_use_two_wires == 0}
-		field = "No2 Direction #1",						etb2_directionPin1, {throttlePedalPositionAdcChannel != 16}
-		field = "No2 Direction #2",						etb2_directionPin2, {throttlePedalPositionAdcChannel != 16}
+		field = "No1 Direction #1",						etbIo1_directionPin1, {throttlePedalPositionAdcChannel != 16}
```

Comment:
you are making a huge mistake here. Your "boost control" draft is messing with ETB settings.

Reply:
I can change them back, but the old pin-names doesnt work anymore..